### ISSUE TITLE
[Idea] Being able to connect monitors to text screens to show the text/sprites

### ISSUE THREAD

**Issue by Xenoprophet-Nail** [tsunamayo/Starship-EVO#2927]
Since Monitors can display data from Rails, Rotators, Hinges, and shields. I was thinking "what if we could have them connect to the text screen and display what's there?" We could do a lot more with decorating our ships. Hopefully it's not to crazy of an idea..

**Comment by TIKIRobo** [comment]
OOooo yep that make sense and could help in lots of builds 
(also maybe like what if we could hit F on the screen and then click what we want to to display so it could display text, shields, HP, damaged systems, ect but also if you say rotator/rail progress you then link it to a rotator/rail)

**Comment by Xenoprophet-Nail** [comment]
It already reads progress across the moving parts which is great. But that's all it displays, a lot of people want live video on them which is taxing on any system, that's why I think reading a screen with custom text may be simpler but just as cool.

**Comment by Kadzar** [comment]
I don't really see much point in linking directly to a screen versus just being able to use monitors like we use displays, unless there's some way we could switch between displaying different screens via logic, in which case it might be more practical to have some kind of small logic brick that can contain the text instead of a screen (something similar to the warning message logic brick, or possibly another use for it.)

**Comment by Xenoprophet-Nail** [comment]
ooh, that'd be a cool addition too I like that

**Comment by ZachZent** [comment]
Also as an aside for screens, there have been a few new players asking how to add the sprites to the screens. <sprite=001> isn't exactly intuitive. Instead, would it be possible if we can click on the sprite on the side and it will auto fill in <sprite=001>.

An extra layer down, <sprite=001> being 12 characters makes spacing out specific designs a bit tough requiring more trial and error. Using the same click and insert for the first part, instead of posting the text it would just insert the symbol directly.

**Comment by Hecateus** [comment]
videos and displays should also have a static/snow mode, and a color-test mode.

**Comment by tsunamayo** [comment]
Yes I want to do a nice refactoring of how it works, with a more unified and more powerful approach. But it is a bit low priority to be honest... Cheers
